apps/settings/js/panels/developer/developer.js

```diff
@@ -96,19 +99,25 @@ define(function(require) {
      * @access private
      * @memberOf Developer.prototype
      */
-    _resetDevice: function d__resetDevice() {
+    _resetDevice: function d__resetDevice(event) {
```

Comment:
It's actually used on line 120 to prevent the toggle from being toggled before the reset is confirmed.